dronegen/mac.go

```diff
@@ -517,34 +517,28 @@ func darwinConnectBuildCommands() []string {
 	return commands
 }
 
-func darwinTagCopyPackageArtifactCommands(b buildType, hasTeleportConnect bool) []string {
+func darwinTagCopyPackageArtifactCommands() []string {
```

Comment:
`darwinTagPipeline` no longer builds the Connect artifact so this function no longer needs to care about that artifact.